src/es.jl

```diff
@@ -13,34 +13,35 @@ The constructor takes following keyword arguments:
 - `λ`/`lambda`: the number of offspring
 - `selection`: the selection strategy `:plus` or `:comma` (default: `:plus`)
 """
-struct ES <: AbstractOptimizer
+struct ES{F1,F2} <: AbstractOptimizer
     initStrategy::AbstractStrategy
     recombination::Function
     srecombination::Function
-    mutation::Function
-    smutation::Function
+    mutation::F1
+    smutation::F2
     μ::Integer
     ρ::Integer
     λ::Integer
     selection::Symbol
-
-    ES(; initStrategy::AbstractStrategy = NoStrategy(),
-        recombination::Function = first,
-        srecombination::Function = first,
-        mutation::Function = ((r,s) -> r),
-        smutation::Function = identity,
-        μ::Integer = 1,
-        mu::Integer = μ,
-        ρ::Integer = μ,
-        rho::Integer = ρ,
-        λ::Integer = 1,
-        lambda::Integer = λ,
-        selection::Symbol = :plus) =
-        new(initStrategy, recombination, srecombination, mutation, smutation,
-            mu, rho, lambda, selection)
 end
+
+ES(; initStrategy::AbstractStrategy=NoStrategy(),
+recombination::Function=first,
+srecombination::Function=first,
+mutation::Function=((r,s) -> r),
+smutation::Function=identity,
+μ::Integer=1,
+mu::Integer=μ,
+ρ::Integer=μ,
+rho::Integer=ρ,
+λ::Integer=1,
+lambda::Integer=λ,
+selection::Symbol=:plus) =
```

Comment:
Indentation missing. Same for other constructors.

Reply:
My apologies. The auto formatter in VS Code did not work as expected. It should be resolved in the most recent commit.